source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -899,46 +899,98 @@ Mainnet
 Testnet
 -------
 
-    August 9, 2023
+    October 10, 2023
 
-        Concordium Node 6.0.4 contains support for `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ with Concordium BFT consensus which will be released August 21, 2023. **Node runners should upgrade to version 6.0.4. before the protocol update to ensure that their nodes do not shut down.**
+    Concordium Node 6.1.6 contains bug fixes and improvements.
```

Comment:
We should add a notice here as well saying that this is the last release of the node that has support for V1 gRPC API. In bold.